protocol/src/didcomm/encrypted.rs

```diff
@@ -167,7 +167,7 @@ async fn verify<R: DidRepository>(
         .await
         .map_err(DidCommEncryptedServiceVerifyError::SidetreeFindRequestFailed)?
         .ok_or(DidCommEncryptedServiceVerifyError::DidDocNotFound(
-            other_did,
+            other_did.to_string(),
```

Comment:
```suggestion
            other_did.into_inner(),
```
こちらも同じく. パフォ変わらないならどっちでもだいじょうぶです.